src/scss/pages/_homePage.scss

```diff
@@ -0,0 +1,337 @@
+    margin-left: auto;
+    padding-right: 15px;
+    padding-left: 15px;
+  }
+
+  &-image {
+    &-split {
+      width: 100%;
+      height: auto;
+      display: flex;
+    }
+
+    &-left {
+      max-width: 760px;
+      width: 80%;
+      padding: 0 15px;
+      height: 100%;
+    }
+
+    &-right {
+      width: 20%;
+      height: 100%;
+      padding: 0 15px;
+    }
+  }
+
+  &-left-pics {
+    display: flex;
+  }
+}
+
+.Gaming {
+  &-container {
+    width: 1140px;
+    margin-right: auto;
+    margin-left: auto;
+    height: 270px;
+  }
+
+  &-review {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &-name {
+      color: white;
+    }
+
+    &-text {
+      color: white;
+      text-align: center;
+      max-width: 480px;
+    }
+  }
+
+  &-rating {
+    padding: 0;
+    color: aqua;
+    list-style: none;
+    display: flex;
+  }
+}
+
+.Pricing {
+  &-container {
+    width: 1140px;
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  &-padding {
+    max-width: 33.33%;
+    padding: 0 15px;
+  }
+
+  &-packages {
+    display: flex;
+    justify-content: center;
+  }
+
+  &-background {
+    color: #f2f2f2;
+    background: linear-gradient(to right, #231a60 0%, #3c338d 51%, #231a60 100%);
+    margin-bottom: 40px;
+    width: 350px;
+  }
+
+  &-package-title {
+    text-align: center;
+    margin: 0;
+    padding: 35px 30px 25px;
+  }
+
+  &-price {
+    font-size: 36px;
+    padding: 30px;
+    text-align: center;
+    background: #342b7e;
+  }
+
+  &-list {
+    list-style: none;
+    padding: 35px 50px 18px;
+
+    &-item {
+      font-weight: 400;
+      padding: 14px 0;
+      border-bottom: 1px dashed rgba(255, 255, 255, 0.2);
+    }
+  }
+
+  &-button {
+    max-width: 198px;
+    width: 100%;
+    padding-bottom: 50px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &-tag {
+    text-decoration: none;
+    border: 2px solid;
+    border-image: linear-gradient(to right, #3fcaff 0%, #a4ffb0 51%, #3fcaff 100%);
+    border-image-slice: 1;
+    color: #fff;
+    display: inline-block;
+    overflow: hidden;
+    padding: 0 60px;
+    line-height: 46px;
+
+    &:hover {
+      background: linear-gradient(to right, #3fcaff 0%, #a4ffb0 51%, #3fcaff 100%);
+      color: black;
+    }
+  }
+}
+
+.Question {
+  &-center {
+    width: 1140px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &-main-title {
+    text-align: center;
+    margin-bottom: 60px;
+  }
+
+  &-container {
+    display: flex;
+    border-bottom: 2px dotted grey;
+
+    &-left {
+      max-width: 50%;
+      width: 100%;
+      border-right: 2px dotted grey;
+    }
+
+    &-right {
+      max-width: 50%;
+      width: 100%;
+      margin-left: 80px;
+    }
+
+    &-inner {
+      width: 100%;
+      max-width: 470px;
+    }
+  }
+
+  &-sub-titles {
+    margin: 0;
+    padding-top: 40px;
+  }
+
+  &-text {
+    padding: 30px 20px 40px 0;
+  }
+
+  &-last-border {
+    border-bottom: none;
```

Comment:
Please check if bootstrap have a class for this

Reply:
👍🏼